src/main/java/bio/terra/cli/serialization/userfacing/UFWorkspaceLight.java

```diff
@@ -59,6 +64,11 @@ public void print() {
     OUT.println("Name:              " + name);
     OUT.println("Description:       " + description);
     OUT.println("Google project:    " + googleProjectId);
+    OUT.println("Properties:");
+    for (int counter = 0; counter < properties.size(); counter++) {
```

Comment:
I don't think we need to print counter. Please do this comment?

Reply:
Hi Melissa. I also used this way before. But it does not work. because the properties type is properties. 
![image](https://user-images.githubusercontent.com/100877272/176039971-576c5b76-9e96-4cf5-b47b-550bf9db2a9d.png)
I am not sure why it does not allow. 

I find the reason... I should do 
    for (Property property : properties) {
      OUT.println(
          " " + property.getKey() + ": " + property.getValue());
    }
    
    to specialize the property's type is property